bundles/org.openhab.binding.haywardomnilogic/README.md

```diff
@@ -95,15 +96,17 @@ Hayward OmniLogic Connection Parameters:
 
 | Channel Type ID | Item Type | Description   | Read Write |
 |-----------------|-----------|---------------|:----------:|
-| heaterState     | Number    | Heater state  |      R     |
+| heaterState     | String    | Heater state  |      R     |
```

Comment:
This seems to make the code not backward compatible. You could add a note here https://github.com/openhab/openhab-distro/blob/main/distributions/openhab/src/main/resources/bin/update.lst

Reply:
Is there any documentation available on the update.lst syntax?

Reply:
Not that I'm aware of, but I believe it's not too hard to understand.